bl2/bl2_el3.ld.S

```diff
@@ -174,12 +207,32 @@ SECTIONS
     __RW_END__ = .;
     __BL2_END__ = .;
 
+#if BL2_AT_EL3_ROM
+    __BL2_RAM_START__ = ADDR(.data);
+    __BL2_RAM_END__ = .;
+
+    __DATA_ROM_START__ = LOADADDR(.data);
+    __DATA_SIZE__ = SIZEOF(.data);
+
+    /*
+     * The .data section is the last PROGBITS section so its end marks the end
+     * of BL2's actual content in Trusted ROM.
```

Comment:
minor: Perhaps say "... of BL2's RO content in XIP memory." here.

Reply:
Done, thanks.